transport/grpc/outbound.go

```diff
@@ -217,15 +218,15 @@ func (o *Outbound) invoke(
 			responseBody,
 			callOptions...,
 		),
-	)
+		yarpcerrors.FromError(err).Code())
```

Comment:
You'll get err code from line 208, not from 212.
grpcPeer.clientConn.Invoke should be extracted from UpdateSpanWithErr to have updated err value.